ext/stringio/stringio.c

```diff
@@ -1153,10 +1153,12 @@ strio_each_char(VALUE self)
 
 /*
  * call-seq:
- *   strio.each_codepoint {|c| block }  -> strio
- *   strio.each_codepoint               -> anEnumerator
+ *   each_codepoint {|codepoint| ... } -> self
```

Comment:
Minor: Here, the arrow is `->`.

Reply:
Fixed.  (When copying from HTML, I sometimes get burned by this, and by "smart quotes.")